test/pay/stripe/subscription_test.rb

```diff
@@ -47,6 +47,14 @@ class Pay::Stripe::SubscriptionTest < ActiveSupport::TestCase
     assert_not_nil pay_subscription.ends_at
   end
 
+  test "it will throw an error if the passed argument is not a string" do
+    @user.processor_id = nil
+    @user.card_token = "pm_card_visa"
+
+    plan_hash = {}
+    assert_raise(Pay::Stripe::Error) { @user.subscribe(name: "default", plan: plan_hash) }
```

Comment:
This should test the `swap` method and it currently doesn't.

Should be more like
```
@user.processor = stripe
@user.card_token = "pm_card_visa"
@user.subscribe
assert_raises ArgumentError do
  @user.swap({invalid: :object})
end
```